### PR TITLE
Community search: remove search records view

### PR DESCRIPTION
**Pull request by Pineirin**
Remove community menu.
closes: inveniosoftware/invenio-communities#408

### CODE REVIEW

File: invenio_communities/config.py [hunk 1]
```diff
@@ -105,6 +105,11 @@
 }
 """Community roles."""
 
+COMMUNITIES_RECORDS_SEARCH = {
```
Comment (ntarocco) [inline invenio_communities/config.py:108]: This should not be here, but where the view is defined.

File: invenio_communities/searchapp.py [hunk 2]
```diff
@@ -18,15 +18,13 @@ def search_app_context():
     return {
         # The Following functions should be defined for the communities
         # search_app_communities_config
-        # search_app_communities_records_config
```
Comment (ntarocco) [inline invenio_communities/searchapp.py:21]: can you please check all the comments here and clean them up or report if something is missing?

File: invenio_communities/templates/semantic-ui/invenio_communities/details/header.html [hunk 3]
```diff
@@ -78,15 +78,9 @@ <h2 class="ui header">{{ community.metadata.title }}</h2>
...
-    {% set header_menu_items = {
-      'search': (_('Search'), url_for('invenio_communities.communities_detail', pid_value=community.id), 'search icon', True),
-      'requests': (_('Requests'), url_for('invenio_communities.communities_requests', pid_value=community.id), 'comments icon', permissions.can_search_requests),
-      'members': (_('Members'), url_for('invenio_communities.members', pid_value=community.id), 'users icon', permissions.can_read),
-      'settings': (_('Settings'), url_for('invenio_communities.communities_settings', pid_value=community.id), 'settings icon', permissions.can_update),
-    } %}
     <div class="ui container secondary pointing menu page-subheader pl-0">
-      {% for header_menu_key, (header_menu_item, header_menu_link, menu_icon, permission_to_show ) in header_menu_items.items() if permission_to_show %}
-        <a class="item {{ 'active' if active_header_menu_item == header_menu_key }} {{ 'disabled' if not header_menu_link }}" href="{{ header_menu_link }}"><i aria-hidden="true" class="{{menu_icon}}"></i>{{ header_menu_item }}</a>
+      {% for item in communities_menu if item.permissions or permissions[item.permissions] %}
```
Comment (ntarocco) [inline invenio_communities/templates/semantic-ui/invenio_communities/details/header.html:82]: Suggestion:
```suggestion
      {% for item in communy_details_menu if item.permissions or permissions[item.permissions] %}
```

File: invenio_communities/views/ui.py [hunk 4]
```diff
@@ -125,6 +120,34 @@ def register_menus():
             order=3,
         )
 
+    # @app.context_processor
```
Comment (ntarocco) [inline invenio_communities/views/ui.py:123]: remove?

File: invenio_communities/searchapp.py (hunk 2 above)
Comment (Pineirin) [reply invenio_communities/searchapp.py:21]: I will create an issue for the one that is missing.

**Review by zzacharo — APPROVED** [review]

**Pull request merged**